packages/core/admin/server/services/permission/permissions-manager/sanitize.js

```diff
@@ -256,13 +256,16 @@ module.exports = ({ action, ability, model }) => {
   };
 
   const getQueryFields = (fields = []) => {
+    const nonVisibleAttributes = getNonVisibleAttributes(schema);
+
     return uniq([
       ...fields,
       ...STATIC_FIELDS,
       ...COMPONENT_FIELDS,
       CREATED_AT_ATTRIBUTE,
       UPDATED_AT_ATTRIBUTE,
       PUBLISHED_AT_ATTRIBUTE,
+      ...nonVisibleAttributes,
```

Comment:
At first glance, it seems like nonVisibleAttributes should already be included in `fields` but they aren't, so the better fix might be to find out why line 158 `permittedFieldsOf` isn't including them